src/Sylius/Behat/Resources/config/suites/ui.yml

```
@@ -42,6 +42,7 @@ imports:
     - ui/payment/managing_payment_methods.yml
     - ui/payment/managing_payments.yml
     - ui/product/navigating_between_product_show_and_edit_pages.yaml
+    - ui/payment_request/payment_request_notify.yml
```

Should be added according to the alphabetical order
```suggestion
    - ui/payment_request/payment_request_notify.yaml
```

---

@GSadee All the other files are yml and I think a more large work as to be done here on the entire repository